### PR TITLE
Spport for defaults and local references

### PR DESCRIPTION
**Pull request by DadaDaMotha**
Hi folks,
This is by no means a completion of the features but a step in this direction.

### CODE REVIEW

**Comment by trollfot** [comment]
Thank you very much for your cleanup and work pushing this forward. I'll take time to review it and we can maybe discuss it as we go.

File: src/jsonschema_wtforms/field.py [hunk 1]
```diff
@@ -247,14 +249,22 @@ def from_json_field(
...
         fields = {}
+        definitions = params.get('definitions', {})
         for property_name, definition in properties.items():
             if property_name not in include:
                 continue
-            if (type := definition.get('type', None)) is not None:
-                field = converter.lookup(type)
+            if ref := definition.get('$ref'):
+                if not definitions:
+                    raise NotImplementedError('Missing definitions.')
```
Comment (trollfot) [inline src/jsonschema_wtforms/field.py:271]: Line 178, we have an assertion for  this, here a NotImplementedError. We might want to harmonize it.

File: src/jsonschema_wtforms/field.py [hunk 2]
```diff
@@ -71,7 +71,7 @@ def extract(cls, params: dict, available: set):
                 if 'contentMediaType' in available:
                     ctype = params['contentMediaType']
                     if isinstance(ctype, (list, tuple, set)):
-                        ctype = ','.join(ctype)
+                        ctype = '|'.join(ctype)
```
Comment (trollfot) [inline src/jsonschema_wtforms/field.py:77]: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept advocates for comma separated values. What did motivate this change ?

Comment (DadaDaMotha) [reply src/jsonschema_wtforms/field.py:77]: To pass this test: https://github.com/novareto/jsonschema_wtforms/blob/0ff085d9594814ea33fe0e2a23cf4966c9ed1851/tests/fields/test_array.py#L127

**Pull request merged**